test/onnx/test_fx_op_consistency.py

```diff
@@ -352,6 +352,10 @@
         dtypes=(torch.uint8, torch.int8, torch.int16,),
         reason=onnx_test_common.reason_onnx_script_does_not_support("Add", "int8, int16"),
     ),
+    xfail(
+        "nn.functional.adaptive_avg_pool1d",
+        reason=onnx_test_common.reason_onnx_script_does_not_support("aten::div.Tensor_mode"),
```

Comment:
Plz create an issue under onnxscript for missing ops.. There is at least one model in benchmark needing it too. Thx!

Reply:
https://github.com/microsoft/onnxscript/issues/980

Reply:
This was merged, so remove the `xfail`?

Reply:
Sure, I can update onnx-script version.

Reply:
Tested. Found that we need onnx-script type promotion to support on this.

Reply:
https://github.com/microsoft/onnxscript/issues/563